### ISSUE TITLE
[BUG] - Missing image file and imagenet class index file

### ISSUE THREAD

**Issue by jingedawang** [pytorch/tutorials#2881]
### Add Link

https://pytorch.org/tutorials/beginner/introyt/captumyt.html

### Describe the bug

This tutorial needs a `cat.jpg` file and a `imagenet_class_index.json` file under `img` folder. The author said:
> The place where you got this interactive notebook should also have an img folder with a file cat.jpg in it.

But I didn't find it in the GitHub repo nor the Colab notebook file folder.

I can help to add them in a PR. But to make it align with the video, I'd suggest to find the original image that used in the video, which will be more friendly for learners. Thank you!

### Describe your environment

Google Colab

**Comment by chrisaccount** [comment]
Hi, i had some problem.  found the files by going to the video in Youtube, the description has a couple link.  the first one provides a zip file with files you are seeking.

**Comment by jingedawang** [comment]
> Hi, i had some problem. found the files by going to the video in Youtube, the description has a couple link. the first one provides a zip file with files you are seeking.

Yes, you are right. Thanks for pointing it out. I think it's better to explicitly add this link to the tutorial.
